actions/marketplace.js

```diff
@@ -22,6 +22,10 @@ export function fetchPlugins() {
 
             return {plugins};
         } catch (error) {
+            // If the marketplace server is unreachable, try to get the local plugins only.
+            if (error.server_error_id === 'app.plugin.marketplace_server.app_error' && !localOnly) {
+                await dispatch(fetchPlugins(true));
```

Comment:
and if this fails i guess we'll show still show the error message but with no plugins, right?

Reply:
should we differentiate between reaching remote marketplace and the mm server marketplace api?

Reply:
> and if this fails i guess we'll show still show the error message but with no plugins, right?

Correct. 

> should we differentiate between reaching remote marketplace and the mm server marketplace api?

Where? In what way?

Reply:
@marianunez the text we show in the banner. For when the API fails vs API works but fails to fetch remote plugins.

Reply:
Yes, right now it is showing the [same client side message](https://github.com/mattermost/mattermost-webapp/blob/master/components/plugin_marketplace/marketplace_modal.js#L242-L254) for all errors. We could change it to show the server message directly there so it distinguish what type of error is happening. Thoughts?